components/MainForm.js

```diff
@@ -1,49 +1,80 @@
 import React from 'react'
 import styled from 'styled-components'
 
-const Body = styled.body`
+const Container = styled.div`
+//for mobile
   color: purple;
-  font-size: 1.5em;
   border-radius: 5px;
   text-align: center;  
   font-family: verdana;
-  display: grid;
-  grid-template-rows: repeat(1, 1fr);
-  grid-template-columns: repeat(1, 1fr);
-  grid-gap: 2em;
+  font-size: 2em;
+
+//for tablets
+  @media only screen and (max-width: 425px) and (min-width: 768px) {
+    font-size: 1.5em;
+  }
+
+//for desktops
+  @media only screen and (min-width: 768px) {
+    font-size:1m;
+  }  
 `
 
 const CharacterInput = styled.input.attrs({
   // we can define static props
   type: 'password',
-  placeholder: 'Pasta your password here',
+  placeholder: 'Paste your password here',
 
   // or we can define dynamic ones
   margin: props => props.size || '10px',
-  padding: props => props.size || '10px'
+  padding: props => props.size || '10px',
```

Comment:
I know there is Prettier config or guidelines as such but my preference is no comma dangle